src/pages/Result.vue

```diff
@@ -25,11 +23,25 @@
 export default {
     name: 'Result',
     components: {},
-    props: ["results"],
+    props: ["results", "answers"],
+    methods: {
+      checkTags(answers, tags) {
```

Comment:
I think this should be a computed rather than a method. Computed data is cached by Vue internlly